### DISCUSSION TITLE
How to add section similar to hero in pages or articles

### DISCUSSION THREAD

**Discussion by franckbertuzzi** [GetPublii/Publii#1814] [Help]
Hello
I would like to add in my pages or articles **responsive** sections with title text and image (see screen shot exemple) for showing services.
How can I do that ?
Thank you by advance !
![aide](https://github.com/user-attachments/assets/dfda72c1-72a2-4799-9a4e-fd1b3f8f23e6)

**Comment by candidexmedia** [comment]
Which editor are you using?

**Comment by franckbertuzzi** [comment]
Hello
Visual studio code or Pinegrow

**Reply by candidexmedia** [reply]
Sorry, I meant which Post editor

**Comment by franckbertuzzi** [comment]
Hello
I use WYSIWYG as post editor

**Comment by Vinc26** [comment]
Hi,

I suggest you create two blocks, one for the image, and one with the text ; giving them an ID each. Then adding a custom CSS to layout them.

<img width="1143" alt="Capture d’écran 2024-11-30 à 09 55 53" src="https://github.com/user-attachments/assets/a0c008cb-ab16-43c4-933c-e4434d8773e4">

**Reply by franckbertuzzi** [reply]
Thank you very much 
I will try it tomorrow...
All the best
